### PR TITLE
[Improve](regresstest) update collect distinct regress test for array hash

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx
this regress sql can make sense of  array hashing function is working fine 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

File: regression-test/suites/query_p0/join/test_collect_list_distinct.groovy [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        );
+    """
+
+    sql """insert into ${table3} values (2, 1, "12");"""
+    sql """insert into ${table2} values (1, 7, "12", "car", "amory is cute"); """
+    sql """insert into ${table1} values (7, 30, 1, "amory clever"); """
+    sql """insert into ${table1} values (1, 90, NULL, "car");"""
+    sql """insert into ${table3} values (1, 2, "12");"""
+
+    qt_select_tt  """ select * from ${table3};"""
```
Comment (jacktengg) [inline regression-test/suites/query_p0/join/test_collect_list_distinct.groovy:88]: Better add order by

Comment (amorynan) [reply regression-test/suites/query_p0/join/test_collect_list_distinct.groovy:88]: make sense! done

**Comment by amorynan** [comment]
run buildall

**Review by jacktengg — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**